src/third-party.md

```diff
@@ -50,7 +60,7 @@ Snap it into your existing workflows with the click of a button, automate away t
 * [Apache Cassandra cassandra-sidecar](https://github.com/apache/cassandra-sidecar): Sidecar for the highly scalable Apache Cassandra database, built as part of the Apache Cassandra project.
 * [DataStax Management API for Apache Cassandra](https://github.com/datastax/management-api-for-apache-cassandra): RESTful / Secure Management Sidecar for Apache Cassandra
 * [DataStax Spring Boot](https://github.com/datastax/spring-boot): Spring Boot extension
-* [Instaclustr cassandra-sidecar](https://github.com/instaclustr/cassandra-sidecar): This repository is home of a sidecar for Apache Cassandra database. Sidecar is meant to be run alongside of Cassandra instance and sidecar talks to Cassandra via JMX.
+* [Instaclustr Icarus](https://github.com/instaclustr/icarus): This repository is home of a sidecar for Apache Cassandra database. Icarus is meant to be run alongside of Cassandra instance and it talks to Cassandra via JMX. It embeds [Instaclustr Esop](https://github.com/instaclustr/esop) so you can backup and restore your cluster on-the-fly to and from various cloud storage providers. Icarus is used as a container in [Instaclustr cassandra-operator](https://github.com/instaclustr/cassandra-operator) as well as in [Orange CassKop](https://orange-opensource.github.io/casskop/) where it is used for backup and restoration purposes primarily.
```

Comment:
Could you please tighten up the language? 
Icarus is meant to be run alongside of Cassandra, talking to Cassandra via JMX. [Instaclustr Esop](https://github.com/instaclustr/esop) is embedded in Icarus for on-the-fly cluster backup and restore using various cloud storage providers as a source or destination. Icarus is used primarily as a backup and restore tool, and is containerized in [Instaclustr cassandra-operator](https://github.com/instaclustr/cassandra-operator) and [Orange CassKop](https://orange-opensource.github.io/casskop/).